teps/0089-nonfalsifiable-provenance-support.md

```diff
@@ -463,7 +493,22 @@ Tests for Signed TaskRuns:
 
 ## Design Details - API changes
 
-At the moment, this TEP does not introduce API changes. It stores the necessary data as part of the termination message of TaskRun pods (as results), and TaskRun signatures are included as part of the [embedded `Status` annotation object in the `TaskRun.Status` field](https://github.com/tektoncd/pipeline/blob/302895b5a1ca45c02a85a9822201643c159fe02c/pkg/apis/pipeline/v1beta1/taskrun_types.go#L111). Examples are as shown in the above sections.
+At the moment, this TEP does not introduce any structural API changes. 
+
+### Condition for SignedResultVerified
+
+We add the condition type `SignedResultVerified` as a way for the controller to indicate that the TaskRun pod step results are verified.
+```
+  - lastTransitionTime: '2022-03-14T12:51:00Z'
+    message: Successfully verified all spire signed taskrun results
+    reason: TaskRunResultsVerified
+    status: 'True'
+    type: SignedResultsVerified
+```
```

Comment:
and whether you want to have 2 separate conditions or 1, it would be great to include a list of what the states will mean (e.g. like https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#monitoring-execution-status)

Reply:
We decided on just 1 - sorry for the confusion, we've been going back and forth on this. Big +1 on adding a similar table to describe the status. And will include that in the PR as well!